raaghu-elements/src/rds-alert/rds-alert.stories.tsx

```diff
@@ -105,7 +113,103 @@ export const With_Delay_Alert: Story = {
         iconStroke: true,
         iconHeight: "20px",
         iconWidth: "20px",
+        withBorder: true,
+        displayType: "singleline",
     }
 };
 With_Delay_Alert.parameters = { controls: { include: ['alertmessage', 'colorVariant', 'size', 'dismisable', 'icon', 'iconFill', 'iconStroke', 'iconHeight', 'iconWidth', 'sticky', 'position', 'delay'] } };
 
+export const With_Buttons: Story = {
+    args: {
+        icon: "information",
+        iconFill: false,
+        iconStroke: true,
+        iconHeight: "20px",
+        iconWidth: "20px",
+        linkbutton: true,
+        cancelbutton: true,
+        okaybutton: true,
+        alertheading: "Heading Title: ",
+        alertmessage: "This is the description of the message bar.",
+        colorVariant: "primary",
+        size: "small",
+        dismisable: true,
+        sticky: false,
+        position: "top",
+        withBorder: true,
+        withLeftBorder: false,
+        displayType: "singleline",
+    }
+};
+With_Buttons.parameters = { controls: { include: ['alertmessage', 'icon', 'iconFill', 'iconHeight', 'iconWidth', 'colorVariant', 'size', 'sticky', 'position', 'alertheading', 'withBorder', 'withLeftBorder'] } };
+
+export const With_Buttons_Multiline: Story = {
+    args: {
+        icon: "information",
+        iconFill: false,
+        iconStroke: true,
+        iconHeight: "20px",
+        iconWidth: "20px",
+        linkbutton: true,
+        cancelbutton: true,
+        okaybutton: true,
+        alertheading: "Heading Title: ",
+        alertmessage: "This is the description of the message bar.",
+        colorVariant: "primary",
+        size: "small",
+        dismisable: true,
+        sticky: false,
+        position: "top",
+        withBorder: true,
+        withLeftBorder: false,
+        description: "This is the description which should not exceed 100 character limit.",
+        displayType: "multiline",
+    }
+};
+With_Buttons_Multiline.parameters = { controls: { include: ['alertmessage', 'colorVariant', 'size', 'sticky', 'position', 'alertheading', 'withBorder', 'withLeftBorder', 'description'] } };
+
```

Comment:
@AkshayPotphode passed only required input from agrs to stories remove if not related to that particular story

Reply:
@Ketan-Khalate removed args which is not related to that particular story